components/payments/payment-modal.js

```diff
@@ -20,13 +25,22 @@ export default function AddPaymentModal({ showModal, setShowModal, addNewPayment
           label="Account Number"
           refEl={acctNumInput}
         />
+        <Input 
+          id="expiration"
+          type="text"
```

Comment:
I suggest updating the expiration date input to `type="date"`. This will enforce the correct date format! 
I noticed this because I originally tested with an expiration date of "01-01-2025", and the request came back with a 500 error because it was not formatted as "YYYY-MM-DD". Making the change above fixes that issue.